### PR TITLE
style-guide-link-fix

### PR DESCRIPTION
**Pull request by jmealing**
EN: https://github.com/canada-ca/design-system/pull/453

### CODE REVIEW

File: guide-redaction/index.md [hunk 1]
```diff
@@ -3107,13 +3107,13 @@ title: "Guide de rédaction du contenu du site Canada.ca"
...
+<aside class="wb-fnote" role="note">
 	<h3 id="fn">Notes de bas de page</h3>
 	<dl>
-		<dt id="fn1-dt">Footnote 1</dt>
+		<dt>Footnote 1</dt>
 		<dd id="fn1" tabindex="-1">
-			<p>International Plain Language Federation: <a href="​​https://www.iplfederation.org/plain-language">​https://www.iplfederation.org/plain-language</a></p>
-			<p class="fn-rtn small"><a href="#fn1-0-rf" data-wb-fnote="true"><span class="wb-inv">Return to footnote </span>1<span class="wb-inv"> referrer</span></a></p>
+			<p>International Plain Language Federation:<br><a href="https://www.iplfederation.org/plain-language">​https://www.iplfederation.org/plain-language</a></p>
+			<p class="fn-rtn small"><a href="#fn1-0-rf"><span class="wb-inv">Retour à la note de bas de page </span>1<span class="wb-inv"> referrer</span></a></p>
```
Comment (delisma) [inline guide-redaction/index.md:3116]: Good catch on the English text
```suggestion
			<p class="fn-rtn small"><a href="#fn1-rf"><span class="wb-inv">Retour à la note de bas de page </span>1</a></p>
```

File: guide-redaction/index.md [hunk 2]
```diff
@@ -3107,13 +3107,13 @@ title: "Guide de rédaction du contenu du site Canada.ca"
     <li><a href="https://www.nngroup.com/topic/writing-web/">Writing for the Web</a> (en anglais seulement), Nielsen Norman Group</li>
   </ul>
 </section>
-<aside class="wb-fnote wb-init wb-fnote-inited" role="note" id="wb-auto-4">
+<aside class="wb-fnote" role="note">
 	<h3 id="fn">Notes de bas de page</h3>
 	<dl>
-		<dt id="fn1-dt">Footnote 1</dt>
+		<dt>Footnote 1</dt>
```
Comment (delisma) [inline guide-redaction/index.md:3113]:
```suggestion
		<dt>Note de bas de page 1</dt>
```

File: guide-redaction/index.md [hunk 3]
```diff
@@ -3107,13 +3107,13 @@ title: "Guide de rédaction du contenu du site Canada.ca"
...
 		<dd id="fn1" tabindex="-1">
```
Comment (delisma) [inline guide-redaction/index.md:3114]:
```suggestion
		<dd id="fn1">
```

File: guide-redaction/index.md [hunk 4]
```diff
@@ -3107,13 +3107,13 @@ title: "Guide de rédaction du contenu du site Canada.ca"
...
-			<p>International Plain Language Federation: <a href="​​https://www.iplfederation.org/plain-language">​https://www.iplfederation.org/plain-language</a></p>
```
Comment (delisma) [inline guide-redaction/index.md:3115]: I'm tempted to add _(en anglais seulement)_ but they have that one paragraph in French.

**Review by delisma — CHANGES_REQUESTED** [review]

File: guide-redaction/index.md (hunk 4 above)
Comment (jmealing) [reply guide-redaction/index.md:3115]: Yeah, I had that in my original comments on the working doc I think , but Chelsey was OK with it. We can double check and make another edit if needed

Comment (delisma) [reply guide-redaction/index.md:3115]: No need. If Chelsey already approved it without it. It's a grey area.

**Review by delisma — APPROVED** [review]
🚀 LGTM

**Pull request merged**